client/toDataByLocation.js

```diff
@@ -1,10 +1,10 @@
 export default function toDataByLocation(data) {
   const headers = data.values[1];
-  const approvedIndex = headers.findIndex( e => e === 'approved' );
-  const stateIndex = headers.findIndex( e => e === 'state' );
-  const cityIndex = headers.findIndex( e => e === 'city' );
-  const latIndex = headers.findIndex( e => e === 'lat' );
-  const lngIndex = headers.findIndex( e => e === 'lng' );
+  const approvedIndex = headers.indexOf('approved');
```

Comment:
Shouldn't babel handle this?

Reply:
I'd think so. Still trying to decipher the babel docs regarding polyfills.

Reply:
Think I got it sorted. Will verify with good ol' IE in the morning.

Reply:
you rock. Thank you!!!

Reply:
Turning on the polyfills via Babel does the trick. It really bloats the .entry.js files, though:

![image](https://user-images.githubusercontent.com/743030/79998618-c20ba200-846f-11ea-949e-caf3867850a8.png)